Automattic/jetpack
# Boost: Add post purchase page

Fixes #23319

#### Changes proposed in this Pull Request:

* This adds the landing page after boost was upgraded successfully.
* Improve the design of upgrade page.


#### Jetpack product discussion
pc9hqz-iD-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Go to 'Boost dashboard'
* Add `#/purchase-successful` to the URL
* Check the design on different screen sizes. dVyv3AO1fXG0gprXywMuMh-fi-2130%3A7342

<img width="1475" alt="image" src="https://user-images.githubusercontent.com/3737780/160827083-06de466c-8064-48e4-a4f6-0f79db14a0e1.png">

![image](https://user-images.githubusercontent.com/3737780/161200812-bbe0c3b5-5556-4b90-bb77-34f4abfd553c.png)
